client/src/pages/ForumPage/ForumPage.css

```diff
@@ -38,6 +38,10 @@
   margin: 10px 50px 30px 0px;
 }
 
+a:link {
+  text-decoration: none!important;
+}
```

Comment:
Can you use the css naming convention we discussed? Also, you'll need to find a way to fix this without using `!important`. You have some styles clashing somewhere. Can you debug that?

Reply:
Updated CSS with inline text-decoration fix